trunk/Public/DynamicConditionsPublic.php

```diff
@@ -241,6 +241,10 @@ private function convertAcfDate( $id, array $data ) {
      * @param Locations_Manager $locationManager
      */
     public function checkPopupsCondition( $locationManager ) {
+        if ( Plugin::$instance->preview->is_preview_mode() ) {
```

Comment:
If think we should do it a little bit saver with
```
!empty(Plugin::$instance->preview) && Plugin::$instance->preview->is_preview_mode()) {
```

Reply:
I think it's not needed because the event can't be triggered without elementor 
```php
 $this->loader->addAction( "elementor/theme/before_do_popup", $pluginPublic, 'checkPopupsCondition', 10, 1 );
```
but sure I added it in cae3324

Reply:
But i can be triggered outside of preview. And it is possible, that elementor will remove ->preview if not in preview.